config/vufind/EDS.ini

```diff
@@ -418,6 +418,11 @@ AU = None
 ; useQuery above.
 ;useQueryMinLength = 3
 
+; When using the query string as a match point, the query string and
+; database names will first be normalized by removing the characters
+; in this regular expression. Default is below, use false to disable.
+;useQueryReplacePattern = '/[-\/\.,:]/'
```

Comment:
One last nitpick. The setting and property are called `useQueryReplacePattern`, which implies a boolean to me; should we just call everything `queryReplacePattern` instead?